Nsubjettiness/MeasureDefinition.cc

```diff
@@ -383,9 +383,9 @@ std::vector<LightLikeAxis> DefaultMeasure::UpdateAxesFast(const std::vector <Lig
                                                           ) const {
    assert(old_axes.size() == N);
    
-   // some storage, declared static to save allocation/re-allocation costs
-   static LightLikeAxis new_axes[N];
-   static fastjet::PseudoJet new_jets[N];
+   // some storage, declared on stack to save allocation/re-allocation costs
```

Comment:
I don't see how this comment makes sense anymore. The arrays will be constructed and destroyed every time this method is called, so there will be some performance cost compared to declaring them `static`. The change is necessary for thread-safety, but the comment should be deleted.